src/main/java/com/example/demo/controller/AuthenticationController.java

```diff
@@ -0,0 +1,26 @@
+package com.example.demo.controller;
+
+import com.example.demo.dto.user.UserRegistrationRequestDto;
+import com.example.demo.dto.user.UserResponseDto;
+import com.example.demo.exception.RegistrationException;
+import com.example.demo.service.UserService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+@Validated
+@RequiredArgsConstructor
+public class AuthenticationController {
+    private final UserService userService;
```

Comment:
do you have checkstyle? 
check it please and add if not

Add empty line after field